certified-connectors/AIForged/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4610 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AIForged",
+    "description": "AIForged is an Intelligent Document Processing solution to complex problems.  AIForged can automate the processing and extraction of structured data from unstructured images.  The connector provides integration and automation of intellegent document extraction with AIForged",
```

Comment:
@aiforged Please remove the extra spaces between sentences in the description value.

Reply:
extra spaces removed in latest commit

Reply:
@aiforged  Extra spaces are not removed

Reply:
Removed extra spaces